index.html

```diff
@@ -6118,14 +6205,20 @@ <h3 id="Mortality">Mortality</h3>
             <tr>
                <td>4
                </td>
-               <td>Death Ward
+               <td>Duality
                </td>
-               <td>Mark a target within 100ft, if they are reduced to 0HP they immediately
-                  regain Heavy+SPI HP and do not fall unconscious.
+               <td>
+                  Select 2 points within 100ft, create a 10ft radius zone around each, choose 
```

Comment:
Death Ward is too similar to Don't Die On Me while also being much worse as you have to predict it and enemies can just target someone else. Don't Die On Me could be made to heal instead, I'm not against that, but probably too strong as it can effectively negate a lot of damage.